audio_sync/cli.py

```diff
@@ -74,6 +77,12 @@ def ParseArgs(args):
   parser.add_argument('--latency_threshold', type=float, default=0.001,
                       help=('Latencies equal or greater than this threshold '
                             'are considered excessive.'))
+  parser.add_argument('--plot_ascii_graph', default=False, action='store_true',
+                      help=('Plots all latencies as ASCII art.'))
+  parser.add_argument('--start_time', default='00:00:00',
+                      help=('hh:mm:ss of when playback started.'))
+  parser.add_argument('--dots_per_usec', type=float, default='0.01',
+                      help=('Dots per usecs in ASCI graph.'))
```

Comment:
I think your description above ("how many ASCII dots are used per usec of latency") is clearer. Suggest to use too as the help message.

Actually, it's weird that the value is fractional. Since we're specifying the number of dots per us, I'd spec this to be an integer value.

Reply:
Changed the description.

I switched the unit and name to "dots_per_msec". That one can use integer values.